packages/stylelint-plugin-dialtone/lib/rules/no-mixins.js

```diff
@@ -25,20 +25,15 @@ const ruleFunction = (primary) => {
     if (!validOptions) return;
 
     // This iterates through one selector at a time, so you don't have to worry about checking for nested selectors.
-    root.walkRules((ruleNode) => {
-      const { source } = ruleNode;
-
-      // any line that starts with a period and ends with a semicolon is a LESS mixin.
-      const regex = /\.[\S]+;/gm;
-
-      const matches = [...source.input.css.matchAll(regex)];
-      for (const match of matches) {
+    root.walkAtRules((ruleNode) => {
+      if (ruleNode.mixin) {
```

Comment:
Nice, there's a check to verify if a node is a mixin! can you please share the docs to read more about it please?

Also, It'd be better to use the early return pattern here, to improve readability.

```suggestion
      if (!ruleNode.mixin) return;
```

Reply:
good idea, thanks.

Reply:
docs here: https://github.com/shellscape/postcss-less?tab=readme-ov-file#mixins